app/assets/stylesheets/lux/_show_heading.scss

```diff
@@ -0,0 +1,8 @@
+@import "lux/variables";
+
+div#document.document > div > h1 {
+  font: normal bold 48px/normal "Cardo", serif;
```

Comment:
Could you adapt one of our variables in `_fonts.scss` to achieve this, with rems?

Reply:
This has been addressed and in the next commit.